tests/utils_tests/test_feedgenerator.py

```diff
@@ -159,3 +159,20 @@ def test_stylesheet_keeps_lazy_urls(self):
             str(stylesheet), 'href="test.css" type="text/css" media="screen"'
         )
         m.assert_called_once()
+
+    def test_stylesheet_attribute_escaping(self):
+        """
+        Stylesheet.__str__() should escape attribute values.
+        """
```

Comment:
You can chop this docstring; we favor descriptive method names instead (which you have done).